broker/cloud_functions/lvk/store_BigQuery/main.py

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env python3
+# -*- coding: UTF-8 -*-
+
+"""This module stores alert data in BigQuery tables."""
+
+import base64
+import os
+import pittgoogle
+from google.cloud import functions_v1, pubsub_v1, logging
+
+PROJECT_ID = os.getenv("GCP_PROJECT")
+SURVEY = os.getenv("SURVEY")
+TESTID = os.getenv("TESTID")
+VERSIONTAG = os.getenv("VERSIONTAG")
+
+# connect to the cloud logger
+log_name = "store-bigquery-cloudfnc"  # same log for all broker instances
+logging_client = logging.Client()
+logger = logging_client.logger(log_name)
+
+# GCP resources used in this module
+pittgoogle_project = pittgoogle.ProjectIds().pittgoogle
+MODULE_NAME = f"alerts_{VERSIONTAG}"
+TABLE = pittgoogle.Table.from_cloud(MODULE_NAME, survey=SURVEY, testid=TESTID)
+TOPIC = pittgoogle.Topic.from_cloud("BigQuery", survey=SURVEY, testid=TESTID, projectid=PROJECT_ID)
+
+# 'context' is an unused argument in the function below that is required
+# see https://cloud.google.com/functions/1stgendocs/writing/write-event-driven-functions#background-functions
+def run(event: dict, _context: functions_v1.context.Context) -> None:
+    """Send alert data to various BigQuery tables.
+
+    Args:
+        event: Pub/Sub message data and attributes.
+            `data` field contains the message data in a base64-encoded string.
+            `attributes` field contains the message's custom attributes in a dict.
+
+        context: Metadata describing the Cloud Function's trigging event.
+    """
+    
+    # decode the base64-encoded message data
+    decoded_data = base64.b64decode(event["data"])
+
+    # create a PubsubMessage-like object with the existing event dictionary
+    pubsub_message = pubsub_v1.types.PubsubMessage(
+        data=decoded_data,
+        attributes=event.get("attributes", {})
+    )
+
+    # unpack the alert
+    alert = pittgoogle.Alert.from_msg(msg=pubsub_message,schema_name="default_schema")
```

Comment:
Good for now. I'm in the process of adding a `from_cloud_functions` method that will take care of the decoding, etc. in pittgoogle-client.